styles/semaphore.module.scss

```diff
@@ -0,0 +1,100 @@
+@use 'common/input.module' as input_layout;
+
+@mixin semaphore_button_mixin {
+  position: absolute;
+  //min-width: 60px;
+  //min-height: 60px;
+  display: block;
+  height: 30%;
+  width: 30%;
+
+  & > p {
+    display: none;
+  }
+
+  & > svg {
+    width: 100%;
+    height: 100%;
+  }
+}
+
+.semaphore_button_1 {
+  top: 100%;
+  left: 50%;
+  transform: translate(-50%, -100%);
+  @include semaphore_button_mixin;
+}
+
+.semaphore_button_2 {
+  top: 85%;
+  left: 15%;
+  transform: translate(-25%, -75%);
+  @include semaphore_button_mixin;
+}
+
+.semaphore_button_3 {
+  top: 50%;
+  left: 0%;
+  transform: translate(0%, -50%);
+  @include semaphore_button_mixin;
+}
+
+.semaphore_button_4 {
+  top: 15%;
+  left: 15%;
+  transform: translate(-25%, -25%);
+  @include semaphore_button_mixin;
+}
+
+.semaphore_button_5 {
+  top: 0%;
+  left: 50%;
+  transform: translate(-50%, 0%);
+  @include semaphore_button_mixin;
+}
+
+.semaphore_button_6 {
+  top: 15%;
+  left: 85%;
+  transform: translate(-75%, -25%);
+  @include semaphore_button_mixin;
+}
+
+.semaphore_button_7 {
+  top: 50%;
+  left: 100%;
+  transform: translate(-100%, -50%);
+  @include semaphore_button_mixin;
+}
+
+.semaphore_button_8 {
+  top: 85%;
+  left: 85%;
+  transform: translate(-75%, -75%);
+  @include semaphore_button_mixin;
+}
+
+.buttons_box {
+  display: compact;
+  margin: 0 auto;
+  gap: 2vmin;
+  position: relative;
+
+  width: 81vmin;
```

Comment:
Calculate width and height in a different way for landscape/portrait and screen size to avoid overflow.